corehq/apps/dashboard/views.py

```diff
@@ -177,7 +177,7 @@ def _can_access_reminders(request):
         and request.couch_user.can_edit_data()
     )
 
-    is_domain_admin = lambda request: request.couch_user.is_domain_admin(request.domain)
+    is_billing_admin = lambda request: request.couch_user.has_permission(request.domain, 'billing_admin')
```

Comment:
should we not call the permission `administrate_billing` then? so that the function would be `can_administrate_billing`?